src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage;
+  closeError: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  closeError,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      closeError();
+    }, 3000);
+  }, []);
```

Comment:
Missing dependency 'closeError' in the useEffect dependency array. This will ensure that the timer is reset whenever the closeError function changes. 🕒